src/lib/Components/Lists/SavedShowItemRow.tsx

```diff
@@ -1,38 +1,45 @@
 import { Box, color, Flex, Sans, Separator } from "@artsy/palette"
+import { SavedShowItemRow_show } from "__generated__/SavedShowItemRow_show.graphql"
 import { SavedShowItemRowMutation } from "__generated__/SavedShowItemRowMutation.graphql"
 import OpaqueImageView from "lib/Components/OpaqueImageView"
 import Switchboard from "lib/NativeModules/SwitchBoard"
+import { Schema, Track, track as _track } from "lib/utils/track"
 import React from "react"
 import { TouchableWithoutFeedback } from "react-native"
-import { commitMutation, graphql, RelayProp } from "react-relay"
+import { commitMutation, createFragmentContainer, graphql, RelayProp } from "react-relay"
 
 interface Props {
-  __id: string
-  _id: string
-  id: string
-  href: string
-  name: string
-  status: string
-  images: {
-    url: string
-  }
-  partner: {
-    name: string
-  }
-  is_followed: boolean
-  relayEnvironment: RelayProp
+  show: SavedShowItemRow_show
```

Comment:
By defining our properties in a fragment (below), the Relay compiler will generate the `SavedShowItemRow_show` type for us, which we import above. This is great because we don't have to define our types anymore – they're defined by the Metaphysics schema for us 🎉

Reply:
Nice!